src/index.ts

```diff
@@ -16,10 +17,11 @@ export async function run(): Promise<void> {
         const client : SecretsManagerClient = new SecretsManagerClient({region: process.env.AWS_DEFAULT_REGION, customUserAgent: "github-action"});
         const secretConfigInputs: string[] = [...new Set(core.getMultilineInput('secret-ids'))];
         const parseJsonSecrets = core.getBooleanInput('parse-json-secrets');
+        const nameTransformation = parseTransformationFunction(core.getInput('name-transformation'));
```

Comment:
I may be completely missing something, but don't we need a null check on the name-transformation input here? Or else parseTransformationFunction throws an error right?

Reply:
The result of `getInput(...)` is not nullable (checkout the function definition below). It would need to be `: string | null`

```
/**
 * Gets the value of an input.
 * Unless trimWhitespace is set to false in InputOptions, the value is also trimmed.
 * Returns an empty string if the value is not defined.
 *
 * @param     name     name of the input to get
 * @param     options  optional. See InputOptions.
 * @returns   string
 */
export declare function getInput(name: string, options?: InputOptions): string;
```

It returns `uppercase` when user of the action doesn't specify it ([parameter definition](https://github.com/aws-actions/aws-secretsmanager-get-secrets/blob/feature/name-transformation/action.yml#L15-L18)). 

I actually have [integration tests](https://github.com/aws-actions/aws-secretsmanager-get-secrets/blob/feature/name-transformation/.github/workflows/tests.yml#L99-L119) around this. There is no `name-transformation` in the `with:` section and it still passes the `__integration_tests__/name_transformation/uppercase.test.ts`.